### ISSUE TITLE
deploy front end connected to demo network

### ISSUE THREAD

**Issue by dckc** [rchain-community/rgov#56]
I gather this is in progress; http://rhobot.net:8888/ https://rgov.rhobot.net/ have a **rhobot** network option.

When I first went to https://rgov.rhobot.net/ I ran into:

```
rnode.js:63 Mixed Content: The page at 'https://rgov.rhobot.net/' was loaded over HTTPS, but requested an insecure resource 'http://rhobot.net:40403/api/explore-deploy'. This request has been blocked; the content must be served over HTTPS.
```

Perhaps this part is actually working. But reportedly "everything broke" and it's not clear to me whether the problem was in the back end network (#55) or this one.

@jimscarver @Bill-Keuntje is it clear who has the ball on this? Would you please nominate someone by assigning them?

**Comment by dckc** [comment]
goal: 1 URL that's clearly working.

@jimscarver notes  http://rhobot.net:8888/ is close but lacks SSL.

**Comment by jimscarver** [comment]
ssl is now supported at https://rgov.rhobot,net We are also creating a separate instance at demo.rhobot.net specifically for official demonstrations.

**Comment by dckc** [comment]
seems like we're supposed to use http://demo.rhobot.net:8888/

**Comment by dckc** [comment]
demo rehearsal with http://demo.rhobot.net:8888/ was successful; that's what we plan to use.

**Comment by dckc** [comment]
I note https://rgov1.netlify.app/ is now up-to-date. It doesn't default to the demo rchain network, but it does have SSL.

I expect it's sufficiently reliable for the demo, but since it's not what we rehearsed with, I'm not sure we should switch now.

On the other hand, http://demo.rhobot.net:8888/ isn't up-to-date with master (it still has 2 ToDo lists at the bottom and I suspect it's missing the fix for #57 ).

So _something_ should change.

**Comment by dckc** [comment]
We're at T-2hrs and this is still open. Are we doing the demo
  1. at http://demo.rhobot.net:8888/ without the fix for #57 or
  2. at https://rgov1.netlify.app/ accepting a risk that it's different from rehearsal?

**Comment by dckc** [comment]
we went with  http://demo.rhobot.net:8888/
